0800-appendix-198217-trib-to-skeena.Rmd

```diff
@@ -217,6 +217,14 @@ knitr::include_graphics(my_photo, dpi = NA)
 
 <br>
 
+## Fish Sampling {.unnumbered}
+
+Electrofishing was conducted at crossing `r as.character(my_site2)` in 2023 with results summarised in Tables \@ref(tab:tab-fish-site-198217) - \@ref(tab:tab-fish-dens-198217) and Figure
```

Comment:
this seems to jump right into the discussion of fish sampling in 2023 without introducing that the site was revisited in the intro/background.  Also, I would be surprised if there wasn't some help that could be given to the rest of the text 

Need to note about which fish were pit tagged and where the data describing which tag is which is kept.  Consider using a `param` in the `index.Rmd` `yaml` so that it is can be somewhat consistent between reports (this one will be a bit diff than current b/c we have a `data/2023` directory but still useful for other reports that have multple years of data..

2022
>Electrofishing and PIT tagging upstream and downstream of crossing 198217 (including within the northern tributary that feeds the drinking water intake facility) is recommended to provide insight into fish community composition, density, survival and movement.

Reply:
> Need to note about which fish were pit tagged and where the data describing where tag data is kept. Consider using a param in the index.Rmd yaml so that it is can be somewhat consistent between reports (this one will be a bit diff than current b/c we have a data/2023 directory but still useful for other reports that have multple years of data..

should I make a param specifically for https://github.com/NewGraphEnvironment/fish_passage_skeena_2022_reporting/blob/main/data/2023/habitat_confirmations.xls which contains the pit tag data? Or are u thinking more general such as just the repo_url (like [here](https://github.com/NewGraphEnvironment/fish_passage_skeena_2023_reporting/blob/main/index.Rmd#L27C1-L29C89)) and then we paste the rest of the path?

Reply:
fyi im working on this for site 198215, will leave 198217 for you still (I will add fish data tho).